test/run.d

```diff
@@ -167,7 +167,7 @@ void ensureToolsExists(string[string] env, const TestTool[] tools ...)
             targetBin = resultsDir.buildPath(tool).exeName;
             sourceFile = toolsDir.buildPath(tool ~ ".d");
         }
-        if (targetBin.timeLastModified.ifThrown(SysTime.init) >= sourceFile.timeLastModified)
+        if (1||targetBin.timeLastModified.ifThrown(SysTime.init) >= sourceFile.timeLastModified)
```

Comment:
I assume this was a debugging device?

Reply:
Yeah, thanks for spotting it. Sometimes unwanted things sneak in to pull requests.